src/Traveler.js

```diff
@@ -0,0 +1,44 @@
+class Traveler {
+  constructor({
+    id,
+    name,
```

Comment:
I've always wanted to use this in a project but never have, looks clean and easy to understand.